library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/DashManifestParser.java

```diff
@@ -978,6 +981,25 @@ protected RangedUri buildRangedUri(String urlText, long rangeStart, long rangeLe
     return new RangedUri(urlText, rangeStart, rangeLength);
   }
 
+  protected ProgramInformation parseProgramInformation(XmlPullParser xpp) throws IOException, XmlPullParserException {
+    String title = null;
+    String source = null;
+    String copyright = null;
+    String moreInformationURL = parseString(xpp, "moreInformationURL", null);
+    String lang = parseString(xpp, "lang", null);
+    do {
+      xpp.next();
+      if (XmlPullParserUtil.isStartTag(xpp, "Title")) {
+        title = xpp.nextText();
+      } else if (XmlPullParserUtil.isStartTag(xpp, "Source")) {
+        source = xpp.nextText();
+      } else if (XmlPullParserUtil.isStartTag(xpp, "Copyright")) {
+        copyright = xpp.nextText();
+      }
```

Comment:
When reviewing this change I realized there was a general problem in this parser where we'd potentially still process tags nested inside tags we don't recognize. Like:
```
<ProgramInformation>
  <Title>Blah</Title>
  <SomeCustomThing>
    <Title/>                                      <-- We don't want to be parsing this :)
  </SomeCustomThing>
</ProgramInformation>
```
I fixed this in https://github.com/google/ExoPlayer/commit/d511370338415733a1b9a40a23c3e9837dd5ed7d. Please add an
```
else {
  maybeSkipTag(xpp);
}
```
to the end of this block to fix it here as well. Once that's done I think we're good to merge; thanks!

Reply:
I didn't even consider that something like that is technically in line with the spec, we should be good to go now.